src/Microsoft.DotNet.Build.Tasks/VersionTools/BaseDependenciesTask.cs

```diff
@@ -22,14 +22,12 @@ public abstract class BaseDependenciesTask : BuildTask
         internal const string VersionsRepoDirMetadataName = "VersionsRepoDir";
         internal const string BuildInfoPathMetadataName = "BuildInfoPath";
         internal const string CurrentRefMetadataName = "CurrentRef";
-        internal const string CurrentBranchMetadataName = "CurrentBranch";
         internal const string PackageIdMetadataName = "PackageId";
         internal const string VersionMetadataName = "Version";
-
-        internal const string AutoUpgradeBranchMetadataName = "Version";
+        internal const string DependencyTypeMetadataName = "DependencyType";
 
         [Required]
-        public ITaskItem[] DependencyBuildInfo { get; set; }
+        public ITaskItem[] DependencyInfo { get; set; }
```

Comment:
Are there any targets calling this task that don't exist in BuildTools? If so then changing this will break them.

Reply:
Not that I'm aware of, and searching the `dotnet` org for `UpdateDependencies` and `VerifyDependencies` only shows the usages in the BuildTools targets.